draft-ietf-dprive-dnsoquic.md

```diff
@@ -296,9 +296,9 @@ The server MUST send the response(s) on the same stream and MUST indicate, after
 the last response, through the STREAM FIN mechanism that no further data will be
 sent on that stream.
 
-Therefore, a single client-initiated DNS transaction consumes a single stream.
+Therefore, a single unidirectional client-initiated DNS transaction consumes a single stream.
```

Comment:
I think the reference below is useful - thank you! However the streams used by DoQ are bidirectional (as are DNS transactions) so I don't think `unidirectional` here is correct unless I missed your point?

Reply:
Sorry, you are right. This should be bidirectional. In QUIC there are both unidirectional and bidirectional stream and therefore I thought it's useful to clarify. however, this is probably anyway in the wrong position. Let me try another way...